glyphs2fontir/src/source.rs

```diff
@@ -884,24 +950,39 @@ impl Work<Context, WorkId, Error> for GlyphIrWork {
         let static_metadata = context.static_metadata.get();
         let axes = &static_metadata.all_source_axes;
         let global_metrics = context.global_metrics.get();
+        let is_bracket_layer = self.base_name.is_some();
```

Comment:
This feels a bit obtuse, I wonder if we could at least encapsulate this in a method such that this reads self.is_bracket_layer()?